src/Components/FilterComponents/SelectFirstImportancePoi.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import '../ComponentsCSS/FilterComponent.scss';
+
+const mapStateToProps = state => ({
+  poiKeywordsDisplay: state.pois.poiKeywordsDisplay,
+  displaySecondImportancePoiPage: state.pois.displaySecondImportancePoiPage,
+  firstIndicationIsDisplayed: state.pois.firstIndicationIsDisplayed,
+  filterKeywordPageDisplay: state.pois.filterKeywordPageDisplay,
+});
+
+const selectRightKeywordChildren = (specificKeyword, poiKeywordsDisplay) => poiKeywordsDisplay.filter(keyword => keyword.parent_id === specificKeyword.id);
+
+const onlyKeywordsFirstImportance = keywords => keywords.filter(
```

Comment:
Indent 'selectRightKeywordChildren' and 'onlyKeywordsFirstImportance' the same way

Reply:
indentation changed